docs/maintain/maintain-guides-how-to-monitor-your-node.md

```diff
@@ -249,7 +249,7 @@ sudo systemctl enable grafana-server
 sudo systemctl start grafana-server
 ```
 
-You can now access it by going to the `http://SERVER_IP_ADDRESS:3000/login`. The default user and
+You can now access it by going to the `https://SERVER_IP_ADDRESS:3000/login`. The default user and
```

Comment:
in most cases, a plain server Ip will also not be secure, it is usually behind a URL.

```suggestion
You can now access it by going to the `http://SERVER_IP_ADDRESS:3000/login`. The default user and
```